ui/pages/swaps/build-quote/index.scss

```
@@ -84,17 +84,17 @@
     &--error {
       div:first-of-type {
         font-weight: bold;
-        color: var(--Black-100);
+        color: var(--color-text-default);
       }
 
       .build-quote__form-error:first-of-type {
         font-weight: bold;
-        color: var(--Red-500);
+        color: var(--color-error-default);
       }
 
       div:last-of-type {
         font-weight: normal;
-        color: var(--Grey-500);
+        color: var(--color-text-muted);
```

Not sure if this text is important but I'd ere on the side of caution here and choose the higher contrasting design token. We should assume text muted is not accessible and has very light contrast. Grey500 is kind of in the middle between text-alternative and text-muted.

```suggestion
        color: var(--color-text-alternative);
```